### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by srikrsna-buf** [connectrpc/connect-es#1032] [Announcements]
## What's Changed

This release includes support for server-side interceptors! Here's a quick example:

```ts
import * as http from "http";
import routes from "./connect";
import { connectNodeAdapter } from "@connectrpc/connect-node";
import type { Interceptor } from "@connectrpc/connect";

const logger: Interceptor = (next) => async (req) => {
  console.log(`recevied message on ${req.url}`);
  return await next(req);
};

http
  .createServer(
    connectNodeAdapter({
      routes,
      interceptors: [logger],
    }),
  )
  .listen(8080);
```

For more on them please see the [docs](https://connectrpc.com/docs/node/interceptors).

### Other Changes

* Add service and method in grpc-web unary response by @minimal1 in https://github.com/connectrpc/connect-es/pull/984
* Document the `ts_nocheck` plugin option by @timostamm in https://github.com/connectrpc/connect-es/pull/1012
* Remove node export condition by @smaye81 in https://github.com/connectrpc/connect-es/pull/1017
* Avoid instanceof Message by @timostamm in https://github.com/connectrpc/connect-es/pull/1023

## New Contributors
* @minimal1 made their first contribution in https://github.com/connectrpc/connect-es/pull/984

**Full Changelog**: https://github.com/connectrpc/connect-es/compare/v1.3.0...v1.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/connectrpc/connect-es/releases/tag/v1.4.0'>v1.4.0</a>.</em>

**Comment by slavovojacek** [comment]
I just tested interceptors in my Fastify/Connect project, works like a charm! Thank you 🙏
